inspirehep/modules/theme/jinja2filters.py

```diff
@@ -33,6 +33,7 @@
 
 from flask import session, current_app
 from jinja2.filters import evalcontextfilter
+from inspirehep.utils.jinja2 import render_template_to_string
```

Comment:
Yep, this could be done with one global horizontal PR (and maybe can be automatized? But not so urgent anyway)